doc/api/vm.md

```diff
@@ -125,6 +125,10 @@ console.log(util.inspect(sandbox));
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
+*Note*: Using the `timeout` or `breakOnSigint` options will result in new
+event loops and corresponding threads being started, which have a non-zero
```

Comment:
I'd say something like "Using the `timeout` option will result in creating a new event loop and utilizing the `breakOnSigint` option will cause a helper thread to be started", but, well, I agree that it is an implementation detail and your version is more concise, so I'm fine with it either way.